### ISSUE TITLE
Make sure that text in the name field can't cause errors

### ISSUE THREAD

**Issue by compwron** [rubyforgood/casa#182]
**What type of user is this for? volunteer/supervisor/admin/all**
all

**Description**
Name field: https://casa-r4g-staging.herokuapp.com/users/edit
<img width="516" alt="Screen Shot 2020-04-25 at 2 38 36 PM" src="https://user-images.githubusercontent.com/578159/80291467-89372b80-8702-11ea-8af7-02e7f5e4a0b3.png">

Write tests which put text in the user name field via the UI and make sure that no weird pop-ups happen and no unexpected database commands are issued. Make sure all names work in exported reports.
 
Use https://github.com/minimaxir/big-list-of-naughty-strings/blob/master/naughtystrings/internal/resource.go for inspiration

Do not delete characters from names, just escape them. All of these should be **valid** names. 
1. `مرحبا بالعالم هذا اسم من ترجمة جوجل`
1. `שלום עולם זה שם מגוגל תרגם`
1. `"1'; DROP TABLE users-- 1"`
1. `<<SCRIPT>alert(\"XSS\");//<</SCRIPT>`
1. `Dr. Jane Smith, MS; MST; Esq.`

These characters can be part of names and should be valid also: `؛`  `፤` `꛶`

Basically, everything should be valid, nothing should be dangerous. 

Inspiration: https://github.com/minimaxir/big-list-of-naughty-strings/blob/master/naughtystrings/internal/resource.go

**Screenshots of current behavior, if any**

**Related issues, if any**
